Apps/WebClient/src/ClientApp/src/utility/fragmentStorage.ts

```diff
@@ -83,11 +82,7 @@ export class FragmentedStorage implements Storage {
     }
 
     private getFragmentSection(fragment: string): string {
-        const fragmentSection = fragment.substr(
-            this.fragmentIndicator.length,
-            fragment?.length
-        );
-        return fragmentSection;
+        return fragment.substr(this.fragmentIndicator.length, fragment?.length);
```

Comment:
The downside of the lack of assignment is that debugging is a tad trickier

Reply:
I agree, we can get around the code smell by logging the value if needed for debugging